lib/workers/qemu.js

```diff
@@ -16,12 +16,27 @@
 
 'use strict'
 
+const _ = require('lodash')
 const childProcess = require('child_process')
 
-const childKillEventHandler = (child, event) => {
-  process.on(event, (code) => {
+const manageHandlers = (child, options) => {
+  const action = options.register ? 'once' : 'removeListener'
+
+  // These functions need to be named so we can later remove the listeners
```

Comment:
I believe the issue remains, as each time you call `manageHandlers`, new functions are created, just that they are named now. We basically want `exitFunc` and `signalFunc` to be defined outside `manageHandlers`

Reply:
fixed